src/display/webgl.js

```diff
@@ -83,7 +84,8 @@ var WebGLUtils = (function WebGLUtilsClosure() {
     if (currentGL) {
       return;
     }
-    currentCanvas = document.createElement('canvas');
```

Comment:
revert the change here -- it is used in the webgl context, you can add a comment that CanvasRactory is not needed here